kanidm_client/src/lib.rs

```diff
@@ -68,7 +68,10 @@ impl KanidmClient {
     }
 
     fn build_reqwest(ca: &Option<reqwest::Certificate>) -> Result<reqwest::Client, reqwest::Error> {
-        let client_builder = reqwest::Client::builder().cookie_store(true);
+        let client_builder = reqwest::Client::builder()
+            .cookie_store(true)
+            // .danger_accept_invalid_hostnames(true)
+            .danger_accept_invalid_certs(true);
```

Comment:
Probably should remove this, I think that this was necesitated by a breaking change in reqwest or hyper where the add_root_certificate seems to no longer be respected :(